ShadowWizardMoneyGame/Mob.gd

```diff
@@ -8,6 +15,11 @@ func _ready():
 	$AnimatedSprite.playing = true
 	var mob_types = $AnimatedSprite.frames.get_animation_names()
 	$AnimatedSprite.animation = mob_types[randi() % mob_types.size()]
+	print(_is_Mob_in_Player_Shadow())
```

Comment:
Remove print statements

Reply:
Do this and re-request. Everything else LGTM

Reply:
done